dbt_coves/tasks/generate/base.py

```diff
@@ -76,8 +76,13 @@ def select_schemas(self, schemas):
 
     def get_relations(self, filtered_schemas):
         rel_name_selectors = [
-            relation for relation in self.get_config_value("relations")
+            relation for relation in self.get_config_value("select")
         ]
+
+        rel_excludes = [
+            relation for relation in self.get_config_value("exclude")
+        ]
+
         rel_wildcard_selectors = []
         for rel_name in rel_name_selectors:
             if "*" in rel_name:
```

Comment:
this `if` is not needed

Reply:
Are you sure it's not needed?

```
rel_wildcard_selectors = []
for rel_name in rel_name_selectors:
            if "*" in rel_name:
                rel_wildcard_selectors.append(rel_name.replace("*", ".*"))
```

I think it's managing that only those with wildcard in it's name are added to `rel_wildcard_selectors`

if we remove that `if` all `rel_names` will be added to the list

Reply:
you're right, disregard the comment